### PR TITLE
Add banner to the org readme to improve UX

### PR DESCRIPTION
**Pull request by benjagm**
Add banner to the org readme to improve UX

### CODE REVIEW

File: assets/json-schema-banner.png
Comment (gregsdennis) [inline assets/json-schema-banner.png:1]: Can we adjust the margins to vertically center the text?

Comment (benjagm) [reply assets/json-schema-banner.png:1]: We reviewed the margins and I hope this new versions have fixed it.

**Comment by Relequestual** [comment]
I'd like to see the moto use Title Case, but otherwise it look good.

File: assets/json-schema-banner.png
Comment (gregsdennis) [reply assets/json-schema-banner.png:1]: So much better!

**Comment by gregsdennis** [comment]
> I'd like to see the moto use Title Case, but otherwise it look good.

I disagree.  I like the current casing.

> Build more, break less, empower others.

vs

> Build More, Break Less, Empower Others.

Maybe just the first word in each phrase, with periods?

> Build more. Break Less. Empower others.

**Comment by Relequestual** [comment]
> 
> Maybe just the first word in each phrase, with periods?
> 
> 
> 
> > Build more. Break less. Empower others.

I think I actually prefer this! Nice! I retract my previous ask for this one.

**Comment by benjagm** [comment]
@gregsdennis @Relequestual I updated the banner including your comments.

**Review by gregsdennis — APPROVED** [review]

**Pull request merged**